lab_1_keywords_tfidf/start.py

```diff
@@ -31,6 +34,28 @@
     with open(CORPUS_FREQ_PATH, 'r', encoding='utf-8') as file:
         corpus_freqs = json.load(file)
 
-    RESULT = None
+    clean_text, frequencies, tf_calculated, tfidf_calculated, expected_frequency, \
+        chi_values, significant_words, RESULT = [None for not_undefined in range(8)]
+
+    tokenized_text = clean_and_tokenize(target_text)
```

Comment:
Use CAPS to name the variable outside of functions

Reply:
Done